doc/python_api_reference_vDev.md

```diff
@@ -7546,6 +7547,34 @@ class TableauSimulator:
     """
 ```
 
+<a name="stim.TableauSimulator.__init__"></a>
+```python
+# stim.TableauSimulator.__init__
+
+# (in class stim.TableauSimulator)
+def __init__(
+    self,
+    *,
+    seed: Optional[int] = None,
+) -> None:
+
+    """Initializes a stim.TableauSimulator.
+
+    Args:
+        seed: Optional seed for the pseudo-random number generator.
+
+    Returns:
+        An initialized stim.TableauSimulator.
+
+    Examples:
+        >>> import stim
+        >>> s = stim.TableauSimulator(seed=0)
+        >>> s.h(0)
+        >>> s.measure(0)
+        True
```

Comment:
This test is not actually stable. Replace it by one that makes two simulators and checks for identical outputs.